.platform.app.yaml

```diff
@@ -161,8 +162,11 @@ hooks:
         ##php bin/console doctrine:migrations:migrate --no-interaction --allow-no-migration
         ##php bin/console kaliop:migration:migrate --no-interaction --no-debug
 
-        # When using Solr, there are two cases where you'll need to rebuild indexes:
-        # - When Solr / search configuration changes
+        # Configure ElasticSearch
+        ##php bin/console ezplatform:elasticsearch:put-index-template
```

Comment:
There's an issue here. This gets executed on platform.sh after `composer ezplatform-install` which is followed by `ezplatform:reindex`.
Possibly this command should be moved before `HTTPCACHE_PURGE_TYPE="local" php -d memory_limit=-1 'which composer' ezplatform-install` on line 148. (that worked for QA) \cc @mnocon